tools/clang/tools/dxcompiler/dxcutil.cpp

```diff
@@ -177,8 +171,7 @@ HRESULT ValidateAndAssembleToContainer(AssembleInputs &inputs) {
   std::unique_ptr<llvm::Module> llvmModuleWithDebugInfo;
 
   CComPtr<IDxcValidator> pValidator;
-  bool bInternalValidator = CreateValidator(pValidator, inputs.SelectValidator);
-  // Warning on internal Validator
+  bool bInternalValidator = CreateValidator(pValidator);
```

Comment:
No more return value, internal-only now.  This implies a reworking of subsequent code that depends on `bInternalValidator`.  We should be able to remove complexity here.